internal/v1/handler_oscap.go

```diff
@@ -193,7 +195,7 @@ func (h *Handlers) GetOscapCustomizationsForPolicy(ctx echo.Context, policy uuid
 	return ctx.JSON(http.StatusOK, cust)
 }
 
-func (h *Handlers) lintOpenscap(ctx echo.Context, cust *Customizations, fixup bool, distro Distributions, policy string) ([]BlueprintLintItem, error) {
+func (h *Handlers) lintOpenscap(ctx echo.Context, cust *Customizations, fixup bool, distro Distributions, policy string, blueprintEntry *db.BlueprintEntry) ([]BlueprintLintItem, error) {
```

Comment:
```suggestion
func (h *Handlers) lintOpenscap(ctx echo.Context, cust *Customizations, fixup bool, distro Distributions, policy string, blueprint *db.BlueprintEntry) ([]BlueprintLintItem, error) {
```

I would go even further and use `bpe`, `bp` or just `b` - this is a practice found in Effective Go and review guide: https://go.dev/wiki/CodeReviewComments#variable-names